mypy/test/data.py

```diff
@@ -53,11 +53,11 @@ def parse_test_cases(
             while i < len(p) and p[i].id != 'case':
                 if p[i].id == 'file':
                     # Record an extra file needed for the test case.
-                    files.append((os.path.join(base_path, p[i].arg),
+                    files.append((os.path.join(base_path, p[i].arg or ''),
```

Comment:
IIUC the `''` is unreachable -- when id is `'file'` arg is never None. A different rewrite would be
```
arg = p[i].arg
assert arg is not None
```